BLSQ/iaso
# IA-3206-IA-3207: Move and add projects columns and filter on data source and change request page

Explain what problem this PR is resolving
- Move and add projects columns and filter on data source and change request page
Related JIRA tickets : [IA-3206](https://bluesquare.atlassian.net/browse/IA-3206) [IA-3207](https://bluesquare.atlassian.net/browse/IA-3207)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Add projects filter on data source page and rearrange columns
- Add project filter and column and remove groups column from org unit changes requests

## How to test
- Go to data source page and check if it match the [mockup](https://whimsical.com/multi-project-web-interfaces-modifications-LRbazNKxybyqpULUQiKcJx)
- Go to change requests page and check if it match the [mockup](https://whimsical.com/multi-project-web-interfaces-modifications-LRbazNKxybyqpULUQiKcJx)

## Print screen / video
![Screenshot-from-2024-08-08-17-03-58](https://github.com/user-attachments/assets/2e32a05a-1c7c-4ed4-b962-2ec85bd88b1c)
![Screenshot-from-2024-08-08-17-04-07](https://github.com/user-attachments/assets/42856522-2b88-40c0-b196-92fdcea38051)

## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3206]: https://bluesquare.atlassian.net/browse/IA-3206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IA-3207]: https://bluesquare.atlassian.net/browse/IA-3207?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
